atom/renderer/renderer_client_base.cc

```diff
@@ -170,7 +170,7 @@ void RendererClientBase::RenderViewCreated(content::RenderView* render_view) {
     // If backgroundColor is specified then use it.
     std::string name = cmd->GetSwitchValueASCII(switches::kBackgroundColor);
     // Otherwise use white background.
```

Comment:
Comment is now out of date. I'd just remove the two comments above since the code is obvious.